src/engine/platform/opl.cpp

```diff
@@ -1488,8 +1488,8 @@ void DivPlatformOPL::reset() {
   }
   */
   if (downsample) {
-    const unsigned int downsampledRate=(unsigned int)((double)rate*rate/chipRateBase);
-    OPL3_Reset(&fm,downsampledRate);
+    //const unsigned int downsampledRate=(unsigned int)((double)rate*rate/chipRateBase);
+    OPL3_Reset(&fm,44100);
```

Comment:
No. This exists to allow for OPL4.

Reply:
OPL3-L has also downsampled output (44100hz @ 33.8688MHz); and Nuked-OPL3 resample function is hard-coded to 49716hz base regardless of output rate setting.

Reply:
But doesn't OPL4 resample at 48000Hz?

Reply:
in 48000hz case, OPL4 is resampled twice: 49716(Nuked-OPL3 base rate) -> 44100 then 44100 -> 48000